platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -385,6 +386,12 @@ private void onVideoInit() {
 		GodotEditText edittext = new GodotEditText(activity);
 		edittext.setLayoutParams(new ViewGroup.LayoutParams(LayoutParams.MATCH_PARENT,
 				(int)getResources().getDimension(R.dimen.text_edit_height)));
+		// Prevent GodotEditText from showing on splash screen on devices with Android 14 or newer.
+		// It is already not visible on Android 13 and older versions by default, so there is no
+		// need to do anything when running on these versions.
+		if (Build.VERSION.SDK_INT > Build.VERSION_CODES.TIRAMISU) {
```

Comment:
Let's remove the version check since regardless of the Android version, the intent for the edittext is to never be visible. 
Do keep the first line of the comments though as it explains why we're updating the background color.

Reply:
@m4gr3d, thanks for the review. I've updated this commit & prepared a separate pull request for the master branch (see https://github.com/godotengine/godot/pull/88351).